AuthController.go

```diff
@@ -436,11 +534,15 @@ func (ctl *AuthController) ForgotPassword_Request(c echo.Context) error {
 // ForgotPassword_RequestWithIdentifier - Step 1 to change password
 func (ctl *AuthController) ForgotPassword_RequestWithIdentifier(c echo.Context) (err error) {
 	ctx := c.(*bolo.RequestContext)
+	authPlugin := ctx.App.GetPlugin("auth").(*AuthPlugin)
 
-	ctx.Set("template", "auth/forgot-password-request-with-identifier")
-	mt := c.Get("metatags").(*metatags.HTMLMetaTags)
-	ctx.Title = "Senha perdida - resetar"
-	mt.Title = "Senha perdida - resetar | Monitor do Mercado"
+	isJson := ctx.GetResponseContentType() == "application/json"
+	if !isJson {
+		ctx.Set("template", "auth/forgot-password-request-with-identifier")
+		mt := c.Get("metatags").(*metatags.HTMLMetaTags)
+		mt.Title = "Senha perdida - resetar | Monitor do Mercado"
```

Comment:
This phrase "Monitor do Mercado" is right, since we are working with the go-bolo framework